### ISSUE TITLE
[Bug] file update load feature is not enabled in faqgen

### ISSUE THREAD

**Issue by lianhao** [opea-project/GenAIExamples#1066]
### Priority

P3-Medium

### OS type

Ubuntu

### Hardware type

Xeon-SPR

### Installation method

- [ ] Pull docker images from hub.docker.com
- [X] Build docker images from source

### Deploy method

- [X] Docker compose
- [ ] Docker
- [ ] Kubernetes
- [ ] Helm

### Running nodes

Single Node

### What's the version?

commit id: a3f9811

### Description

Similar to issue #1010, the file upload feature can be found in the faqgen ui, but it's not enabled in the faqgen mega gateway service.

### Reproduce steps

upload a file and generate faq in the faqgen ui

### Raw log

```
[2024-10-31 02:33:59,168] [ INFO] - Base service - Uvicorn server setup on port 8888
INFO: Waiting for application startup.
INFO: Application startup complete.
INFO: Uvicorn running on http://0.0.0.0:8888/ (Press CTRL+C to quit)
[2024-10-31 02:33:59,179] [ INFO] - Base service - HTTP server setup successful
INFO: 192.168.127.216:51168 - "POST /v1/faqgen HTTP/1.0" 200 OK
INFO: 192.168.127.216:58774 - "POST /v1/faqgen/doc_upload HTTP/1.0" 404 Not Found
INFO: 192.168.127.216:34190 - "POST /v1/faqgen HTTP/1.0" 200 OK
INFO: 192.168.127.216:38604 - "POST /v1/faqgen/doc_upload HTTP/1.0" 404 Not Found
INFO: 192.168.127.216:40748 - "POST /v1/faqgen/doc_upload HTTP/1.0" 404 Not Found
INFO: 192.168.127.216:58208 - "POST /v1/faqgen/doc_upload HTTP/1.0" 404 Not Found
```

**Comment by joshuayao** [comment]
Hi @lianhao, it should be supported in v1.1. Can I close this issue?

**Comment by lianhao** [comment]
please paste the PR number (for later tracking purpose) and go ahead to close it. Thx!

**Comment by joshuayao** [comment]
Fixed by opea-project/GenAIComps#866
